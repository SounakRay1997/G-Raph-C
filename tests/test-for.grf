int main() {
    int i;
    int max;
    
    max = 10;
    for (i=0; i < 50; i = i+1) {
        print("poato");
        print(i);
    }
}