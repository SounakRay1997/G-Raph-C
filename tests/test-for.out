poato
0
poato
1
poato
2
poato
3
poato
4
poato
5
poato
6
poato
7
poato
8
poato
9
poato
10
poato
11
poato
12
poato
13
poato
14
poato
15
poato
16
poato
17
poato
18
poato
19
poato
20
poato
21
poato
22
poato
23
poato
24
poato
25
poato
26
poato
27
poato
28
poato
29
poato
30
poato
31
poato
32
poato
33
poato
34
poato
35
poato
36
poato
37
poato
38
poato
39
poato
40
poato
41
poato
42
poato
43
poato
44
poato
45
poato
46
poato
47
poato
48
poato
49
